### ISSUE TITLE
Cannot add my OSX device

### ISSUE THREAD

**Issue by ludo237** [keybase/client#2615]
Hello. I don't know if this is the right place for support like this but the site redirect me here so..

I've installed keybase on windows 10 and on osx. On windows 10 it's fine, it creates the device and the paper key. But when I try to add osx laptop using `keybase device add` it shows me this error after typing my keybase password

```
Verification code: ▶ ERROR No device found for device ID 1aa....
```

But when I type `keybase device list` that particular ID is not listed instead I've my Windows 10 device plus my paper key as it should be.

I also tried to remove the current device id just to be sure, but instead of the expected error it shows me this:

```
$ keybase device remove 1aa.....
▶ ERROR Can't revoke the current device.
```

Does this mean that the current device has an ID but is not linked ? :confounded:

**Comment by ludo237** [comment]
Solved. The problem can be fixed with this steps

- logout from keybase
- uninstall `keybase uninstall`
- remove keybase from bin
- reinstall keybase
- type `keybase login`
- prove your device with passphrase or paper key or whatever
- **use a new name that you never use before** this is important
- enjoy

**Comment by cjb** [comment]
I'd like to reopen this since it seems like you hit a bug somewhere here.

**Comment by maxtaco** [comment]
@ludo237 when you saw that error, were you provisioning via paper key?   Which paper key were you using (what are the first two words?) .  `urban similar...`?

**Comment by maxtaco** [comment]
@ludo237 looks like you reset your account on 4/7.  My guess is you were still trying to use a paper key (`urban similar`) from before the reset.

**Comment by ludo237** [comment]
Nope I was using the new paper key `wood tipical`

**Comment by maxtaco** [comment]
Ah, but from your debug message, it looks like you were using device `1aa...` which was from before you reset your account.  So the bug has something to do with that case.  We'll file a ticket, but it will be low priority for now. Thanks!

**Comment by ludo237** [comment]
Yes indeed I had a previous key with that paper phrase but I did a clean install on my laptop, for other reasons.

@maxtaco just ping me if you need further details. I love keybase and I'd like to help as much as I can

**Comment by maxtaco** [comment]
Thanks @ludo237 that gives us enough to go on!
